operations/node_management/Verify_Node_Removal.md

```diff
@@ -0,0 +1,54 @@
+## Verify Node Removal
+
+Use this procedure to verify that a node has been successfully removed from the system.
+
+### Prerequisites
+
+-   The Cray command line interface \(CLI\) tool is initialized and configured on the system.
+-   This procedure requires the xname of the removed node to be known.
+
+### Procedure
+
+1.  Ensure that the Redfish endpoint of the removed node's BMC has been disabled.
+
+    ```bash
+    ncn-m001# cray hsm inventory redfishEndpoints describe x3000c0s19b4
+    Domain = ""
+    MACAddr = "a4bf012b71a9"
+    UUID = "61b3843b-9d33-4986-ba03-1b8acd0bfd9c"
+    IPAddress = "10.254.2.13"
+    RediscoverOnUpdate = true
+    Hostname = "10.254.2.13"
+    Enabled = false       <<-- The Redfish endpoint has been disabled
```

Comment:
REDS/MEDS No longer mark BMCs as disabled when they go away. So when a node is removed from a system the RedfishEndpoint for the node will remain unchanged. This is related to: https://connect.us.cray.com/jira/browse/CASMHMS-4368

Reply:
However, disabling the redfishendpoint is part of the removal procedure. So it will get disabled manually.